.editorconfig

```diff
@@ -10,7 +10,7 @@ trim_trailing_whitespace = true
 [*.{yaml,yml}]
 indent_size = 2
 
-[{**/*.sql,**/OuterReferenceResolver.md,**gradlew.bat}]
+[{**/*.sql,**/OuterReferenceResolver.md,**gradlew.bat,**/*.parquet,**/*.orc}]
```

Comment:
is this needed, no-one should be editing `.parquet` or `.orc` files by hand afaik?

Reply:
I'm not sure, but the `editorconfig-checker` stage failed prior to adding this (at least for `.orc`, didn't do a separate check for `.parquet`):
https://github.com/substrait-io/substrait-java/actions/runs/13032936614/job/36356268259

Reply:
Weird but okay :D